build/prebuild-kata.sh

```diff
@@ -0,0 +1,8 @@
+# prebuild-kata.sh <kata-folder> <kata-notebook>
```

Comment:
Could you please put this script in the `scripts` folder together with the rest of the build scripts?

Reply:
I placed it in build/ to match the layout we have on all other repos, but also happy to move. Should be done in 4f98e42.

Reply:
I think you renamed `build/` to `scripts/` in the Dockerfile but didn't move the actual file - it still shows as `build/prebuild-kata.sh`, and Binder build fails because it can't find any `scripts/*.sh` files for `chmod`.

Reply:
I did rename it, but forgot to add that to my commit when I pushed. I blame it being Monday. Fixing now.